.circleci/config.yml

```diff
@@ -58,9 +58,13 @@ jobs:
           name: Submitting to open.cdash.org
           working_directory: /build
           command: |
-            env | grep ^CIRCLE > circleci.env
-            ctest -A circleci.env -D ExperimentalSubmit -V
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+              env | grep ^CIRCLE > circleci.env
+              ctest -A circleci.env -D ExperimentalSubmit -V
+            else
+              echo "Skipping open.cdash.org submission..."
```

Comment:
Why skip?  We need to see the dashboard for all branch builds.

Reply:
Hi Hans!  

There's a few reasons to skip cdash - 
- github.com/vxl/vxl doesn't use circleci, so this doesn't affect anything here
- this step is not run if there was a build error, so cdash would never see any error reports
- personally, I'd like to give users the ability to skip cdash submission on their own forks for very experimental feature sets

I could switch this check to a different circleci environment variable (if the environment variable exists and is true, skip cdash) so users are given the opportunity to disable cdash submission if they so desire?